### DISCUSSION TITLE
Status of replication/cluster features?

### DISCUSSION THREAD

**Discussion by assens** [microstream-one/microstream#338] [Q&A]
Hi,
Thank you for your great job. 
Could you give us update on the status of the replication and cluster features?
I'd like to be an easy adopter, if these features are ready for testing.

Cheers,
Assen

**Comment by fh-ms** [comment]
Hi Assen,
these features are currently in alpha, the first integration tests are looking good :)
As soon as it is available for early access, we will announce it on our channels (Twitter, ...)

**Reply by chirdeeptomar** [reply]
Any update on this? Just watched a video here: https://youtu.be/U7t9GrkKKV8

Why do I need an microstream cloud account and run on AWS? Is it not free for me to run on Kubernetes as I wish?
